package.json

```
@@ -24,9 +24,9 @@
     "test:fix": "npm run test:prettier:fix && npm run test:eslint:fix",
     "test:eslint": "eslint 'src/**/*.{js,vue}' --cache",
     "test:eslint:fix": "npm run test:eslint -- --fix",
-    "test:stylelint": "stylelint 'src/**/*.scss' --config node_modules/do-bulma/.stylelintrc.json",
-    "test:i18n-packs": "node --import=extensionless/register src/nginxconfig/i18n/verify.js",
-    "test:prettier": "prettier 'src/**/*.{js,vue}' --check",
+    "test:stylelint": "stylelint 'src/**/*.{scss,css}' --config node_modules/do-bulma/.stylelintrc.json",
+    "test:i18n-packs": "node src/nginxconfig/i18n/verify.js",
+    "test:prettier": "prettier \"src/**/*.{js,vue}\" --check",
```

```suggestion
    "test:prettier": "prettier 'src/**/*.{js,vue}' --check",
```
Any reason to not stick with single quotes here?

---

I apologize for the typo, i was just trying things out, I am using PowerShell in vscode, and i guess it interprets single quotes literally, causing the glob pattern to break...

---

Ah I see -- the quotes are there to stop shells expanding the globs, so that the tool (eslint/prettier/stylelint) expands them in a cross-platform manner. If you're having issues with that, feel free to open a separate PR to tweak what quotes are being used etc.